src/Service.Host/client/src/containers/tariffs/Reallocator.js

```diff
@@ -3,42 +3,40 @@ import {
     getItemErrorDetailMessage,
     initialiseOnMount
 } from '@linn-it/linn-form-components-library';
-import Reallocator from '../../components/salesArticles/Reallocator';
-import salesArticleReallocateActions from '../../actions/salesArticlesReallocate';
-import salesArticlesReallocateSelectors from '../../selectors/salesArticlesReallocateSelectors';
+import Reallocator from '../../components/tariffs/Reallocator';
+import tariffsReallocateActions from '../../actions/tariffsReallocate';
+import tariffsReallocateSelectors from '../../selectors/tariffsReallocateSelectors';
 import tariffsActions from '../../actions/tariffs';
 import tariffsSelectors from '../../selectors/tariffsSelectors';
 import oldTariffsActions from '../../actions/oldTariffs';
 import oldTariffsSelectors from '../../selectors/oldTariffsSelectors';
 import * as itemTypes from '../../itemTypes';
 
 const mapStateToProps = state => ({
-    loading: salesArticlesReallocateSelectors.getLoading(state),
-    errorMessage: getItemErrorDetailMessage(state, itemTypes.salesArticlesReallocate.item),
+    loading: tariffsReallocateSelectors.getLoading(state),
+    errorMessage: getItemErrorDetailMessage(state, itemTypes.TariffsReallocate.item),
     tariffSearchLoading: tariffsSelectors.getSearchLoading(state),
     tariffSearchResults: tariffsSelectors
         .getSearchItems(state)
         .map(s => ({ ...s, id: s.id, name: s.tariffCode, description: s.description })),
-    snackbarVisible: salesArticlesReallocateSelectors.getSnackbarVisible(state),
+    snackbarVisible: tariffsReallocateSelectors.getSnackbarVisible(state),
     tariffSearchErrorMessage: getItemErrorDetailMessage(state, itemTypes.tariffs.item),
     oldTariffSearchLoading: oldTariffsSelectors.getSearchLoading(state),
     oldTariffSearchResults: oldTariffsSelectors
         .getSearchItems(state)
         .map(s => ({ ...s, id: s.id, name: s.tariffCode, description: s.description })),
     oldTariffSearchErrorMessage: getItemErrorDetailMessage(state, itemTypes.oldTariffs.item),
-    item: salesArticlesReallocateSelectors.getItem(state)
+    item: tariffsReallocateSelectors.getItem(state)
 });
 
-const initialise = () => dispatch => {
-    dispatch(salesArticleReallocateActions.fetch());
-};
+const initialise = () => () => {};
```

Comment:
would just ditch this if it's not doing anything and remove the initialiseOnMountCall too